headers/functions/overlay29.h

```diff
@@ -35,7 +35,6 @@ bool MonsterIsType(struct entity* entity, enum type_id type_id);
 bool IqSkillIsEnabled(struct entity* entity, enum iq_skill_id iq_id);
 int GetMovePower(struct entity* entity, struct move* move);
 void AddExpSpecial(struct entity* attacker, struct entity* defender, int base_exp);
-bool PptrIsValid(void** pptr);
 bool DefenderAbilityIsActive(struct entity* attacker, struct entity* defender,
```

Comment:
Hmm, since we're reordering the symbols, we might as well reorder the function signatures as well to keep things nice.

Reply:
Do you mean reordering the signatures to match the order of the functions in the symbol file?

Reply:
Yep, that's what I meant.